lv_objx/lv_list.c

```diff
@@ -712,7 +720,11 @@ static lv_res_t lv_list_signal(lv_obj_t * list, lv_signal_t sign, void * param)
             if(btn != NULL) {
                 lv_action_t rel_action;
                 rel_action = lv_btn_get_action(btn, LV_BTN_ACTION_CLICK);
-                if(rel_action != NULL) rel_action(btn);
+                if(rel_action != NULL) {
+                    lv_list_ext_t * ext = lv_obj_get_ext_attr(list);
+                    ext->last_sel = btn;
```

Comment:
Why save the last_sel only if there is a valid action?
IMO `ext->last_sel = btn;` can go before `if(rel_action != NULL) {`.

Reply:
You're right, oversight on my part. For my application, I just absolutely knew i wanted to save the position right before the callback was triggered. Moved it per your recommendation in 92cb0ac